src/modules/tenant/TenantDropdown.js

```diff
@@ -26,8 +26,19 @@ class TenantDropdown extends Component {
 
 		this.TenantService = props.app.locateService("TenantService");
 
-		this.tenants = this.props.tenants;
+		// Check on tenants loaded from userinfo (array of strings) or from tenant service (array of objects)
```

Comment:
There shouldn't be a difference - it there is, then userinfo implementation is wrong.

Reply:
@ateska userinfo returns a list of strings `tenants = ["tenant1", "tenant2"]` (this is what you will get from API on `/userinfo`), but tenant service returns (I would say more accurate) list of objects `tenants = [{_id:"tenant1"}, {_id:"tenant2"}]` (this is what you will get from API on `/tenants`). Should I communicate that with plum team?

Also, if tenants for authentication should be taken from userinfo, then there is no need to use `/tenants` endpoint, or is it needed for any purpose?

Reply:
After an investigation, `tenants = ["tenant1", "tenant2"]` is the only correct form.